qemu/tests/cfg/hugepage_mem_stress.cfg

```diff
@@ -14,8 +26,15 @@
         install_cmd = "start /wait %s:\HeavyLoadSetup.exe /verysilent"
     variants:
         - @default:
-            extra_params += " -mem-path /mnt/kvm_hugepage"
+            mem-path = /mnt/kvm_hugepage
+            Host_RHEL.m7, Host_RHEL.m8.u0, Host_RHEL.m8.u1, Host_RHEL.m8.u2, Host_RHEL.m8.u3:
+                del mem-path
+                extra_params += " -mem-path /mnt/kvm_hugepage"
         - non_existent_mem_path:
-            start_vm = no
-            not_preprocess = yes
+            mem-path = /mnt/tmp
+            pre_command = "mkdir -p ${mem-path}"
+            post_command = "rm -rf ${mem-path}"
```

Comment:
Unfortunately, it can not work, since the param include  `-` but not `_`.  `mem-path` can not be parse out correctly.

And, `extra_params` use hard code `/mnt/tmp`  which is same with `mem-path`.
To support this, we need involve a new parameter i think.

Reply:
Good idea!  Pls help review again.
 (1/1) type_specific.io-github-autotest-qemu.hugepage_mem_stress.non_existent_mem_path: PASS (119.12 s)